src/main/java/hw/hw11/TextProcessor.java

```diff
@@ -0,0 +1,87 @@
+package hw.hw11;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TextProcessor {
+        public static int countOccurrence(List<String> stringList, String target) {
+        int count = 0;
+        for (String str : stringList) {
+            if (str.equals(target)) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+       public static <T> List<T> toList(T[] array) {
+        List<T> list = new ArrayList<>();
+        for (T item : array) {
+            list.add(item);
+        }
+        return list;
+    }
+
+        public static List<Integer> findUnique(List<Integer> numberList) {
+        List<Integer> uniqueList = new ArrayList<>();
+        for (Integer num : numberList) {
+            if (!uniqueList.contains(num)) {
+                uniqueList.add(num);
+            }
+        }
+        return uniqueList;
+    }
+
+       public static void calcOccurrence(List<String> wordList) {
```

Comment:
такі штуки не тестуються. 

так краще: 
```suggestion
       public static List<String> calcOccurrence(List<String> wordList) {
```